src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,18 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Bulldozer> {
+
+    @Override
+    public List<Bulldozer> get() {
+        List<Bulldozer> list = new ArrayList<>();
+        list.add(new Bulldozer());
+        list.add(new Bulldozer());
+        list.add(new Bulldozer());
```

Comment:
task: "You have abstract class Machine and three sub-classes: Bulldozer, Excavator and Truck. Feel free to add some type-specific fields to these classes." We need to add some type-specific fields to all classes that extends Machine and set some random data to this fields in MachineProducers